src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -685,7 +689,8 @@ private void RobustDelete(string path)
                         }
                         else if (File.Exists(path))
                         {
-                            File.SetAttributes(path, File.GetAttributes(path) & ~FileAttributes.ReadOnly); // make writeable
+                            File.SetAttributes(path, File.GetAttributes(path) & ~FileAttributes.ReadOnly);
+                                // make writeable
```

Comment:
Some refactoring must have moved the comment to a new line. I'll put it back to its original place :)